src/main/ionic/peers/src/app/app.module.ts

```diff
@@ -5,6 +5,7 @@ import { IonicApp, IonicErrorHandler, IonicModule } from 'ionic-angular';
 import { MyApp } from './app.component';
 import { HomePage } from '../pages/home/home';
 import { ListPage } from '../pages/list/list';
```

Comment:
Can you delete the ListPage declarations as well as the actual files?

Reply:
Yeah